README.md

```diff
@@ -1,2 +1,12 @@
-# stylepoint
+# Style Point
 Style Point is a schedule service that connects people to barbers and stylists at the time and location of your choice!
+
+# Status
+[![JavaScript Style Guide](https://img.shields.io/badge/code%20style-standard-brightgreen.svg)](http://standardjs.com/)
+
+# Badges
+[![JavaScript Style Guide](https://cdn.rawgit.com/feross/standard/master/badge.svg)](https://github.com/feross/standard)
+
```

Comment:
Hi @kevinpr007,

Nice job dude! You've brought a huge value to our app. I have only one question. Is the `code style standard` a custom badge? I can't seem to find it in the shields.io website.